spring-cloud-deployer-local/pom.xml

```diff
@@ -43,6 +43,11 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-validation</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate.validator</groupId>
+			<artifactId>hibernate-validator-annotation-processor</artifactId>
+			<version>6.0.2.Final</version>
```

Comment:
Version should go to parent's dependency management. Don't we get any dependency management for this from boot/cloud builds?